src/utils/debug.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * To enable debug messages, enter the following into your
+ * developer tools console:
+ *
+ *   localStorage.debug = 'hellosign-embedded:*';
+ *
+ * This will surface all debug messages under the
+ * "hellosign:" namespace.
+ */
+
+import debug from 'debug';
+
+const info = debug(`${__PKG_NAME__}:info`);
+const warn = debug(`${__PKG_NAME__}:warn`);
+const error = debug(`${__PKG_NAME__}:error`);
+
+info.log = console.log.bind(console); // eslint-disable-line no-console
+warn.log = console.warn.bind(console); // eslint-disable-line no-console
```

Comment:
Is `console.error` already bound by default?

Reply:
Not in this case actually, otherwise when calling `console.log` or `console.warn` it will have the context of the `debug` instance and not the `console`, so it won't work. See here: https://www.npmjs.com/package/debug#output-streams

Reply:
Ah ok!